rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form-original.json

```diff
@@ -114,6 +153,52 @@
           "err": "This field is required"
         }
       },
+      {
+        "key": "doctor_name_label",
+        "type": "label",
+        "text": "Doctor's name",
+        "text_color": "#000000"
+      },
+      {
+        "key": "doctor_name",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "edit_text",
+        "v_required": {
+          "value": "false",
+          "err": ""
+        }
+      },
+      {
+        "key": "doctor_phone_number_lbl",
+        "type": "label",
+        "text": "Doctor's phone number?",
+        "text_color": "#000000"
+      },
+      {
+        "key": "doctor_phone_number",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "edit_text",
+        "v_required": {
+          "value": "false",
+          "err": ""
+        },
+        "v_numeric": {
+          "value": "true",
+          "err": "Value must be numeric"
+        },
+        "v_max_length": {
+          "value": "13",
+          "err": "Phone number cannot be more than 13 digits"
+        },
+        "v_min_length": {
+          "value": "8",
+          "err": "Phone number cannot be less than 8 digits"
+        }
+      },
```

Comment:
I don't think we need this. We should just have the healthworker details.

Reply:
fields removed.